homeassistant/components/mysensors.py

```diff
@@ -495,8 +495,9 @@ def mysensors_callback(msg):
         _LOGGER.debug(
             "Node update: node %s child %s", msg.node_id, msg.child_id)
 
-        child = msg.gateway.sensors[msg.node_id].children.get(msg.child_id)
-        if child is None:
+        try:
+            child = msg.gateway.sensors[msg.node_id].children[msg.child_id]
+        except KeyError:
```

Comment:
child can also be None and that will not check anymore.

Reply:
No child can not be `None`.

Reply:
You are right, I see you replace `.get()`